apps/user/profile/index.coffee

```diff
@@ -61,7 +67,7 @@ module.exports.init = ->
   user = new User USER
 
   view = new ProfileView
-    el: $('.profile')
+    el: $('.UserChannels__groups')
```

Comment:
Pretty please use `js-` scoped *only*. It really makes refactoring very difficult when you're not sure what has js attached to it other wise.